src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.ts

```diff
@@ -137,7 +138,21 @@ export class NavigationComponent implements OnInit, OnDestroy {
 
     return undefined;
   }
+  
+  onMenuClick(event: MouseEvent) {
+    const clickedElement = event.target as HTMLElement;
+    const menuElement = clickedElement.closest('cds-sidenav-menu');
```

Comment:
```suggestion
 const target = event.target;
  if (!(target instanceof HTMLElement)) return;
   const menuElement: Element = target.closest('cds-sidenav-menu');
```
Lets have a typeguard for better safety.

Reply:
Thanks for the suggestion, I just committed the change!